docker-compose/images/kuzzle/Dockerfile

```diff
@@ -4,9 +4,11 @@ RUN  set -x \
   \
   && npm set user 0 \
   && npm set unsafe-perm true \
+  && npm install -g pm2 \
```

Comment:
Why do we need to add pm2 and git to the cluster stack?

Reply:
The image is based on kuzzle's one (not core-dev) to let developers focus on the cluster and not have to worry about mounting kuzzle.
The last image update removed git and pm2, which are required during development, respectively for npm install and for hot-reload.